client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -33,7 +33,21 @@ class SiteTools extends Component {
 	}
 
 	render() {
-		const { translate } = this.props;
+		const {
+			translate,
+			site: { jetpack: isJetpack, is_vip: isVip },
```

Comment:
I'd suggest connecting `SiteTools` and using the `isJetpackSite` and `isVipSite` selectors instead of checking these properties. In combination with `getSelectedSiteSlug`, we even won't need to pass the site to `SiteTools` (and check for `&& site` prior to rendering `SiteTools`).